cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,49 @@
 /// <reference types='cypress' />
+import { dataForRegister } from "../support/dataForRegister";
+import LogIn from "../support/pages/LogIn";
```

Comment:
Is it a login page or a registration page? 
Please use more specific names. For example, logIn => loginPage. The same about Modal (add more details about what it is). Check everywhere